Makefile

```diff
@@ -7,8 +7,8 @@ SHELL := bash
 .ONESHELL:
 
 CERT_MANAGER_VERSION ?= 1.6.2
-BUNDLE_VERSION ?= ${CERT_MANAGER_VERSION}-rc1
-BUNDLE_CHANNELS ?= candidate
+export BUNDLE_VERSION ?= ${CERT_MANAGER_VERSION}
```

Comment:
Exported to make the BUNDLE version available to a new script which generates the PRs for me.